db/migrate/20201030102020_enlarge_report_status.rb

```diff
@@ -0,0 +1,29 @@
+class EnlargeReportStatus < ActiveRecord::Migration[6.0]
+  def up
+    remove_index :reports, :status
+
+    User.as_anonymous_admin do
```

Comment:
This breaks the tests. It is possible that the admin user does not exist at this point. Could we do something like this https://github.com/theforeman/foreman/blob/develop/db/migrate/20170629080604_change_preseed_default_pxe_grub2_kind.rb ?

Reply:
Why do we even need user?

Reply:
> Why do we even need user?

Good one. Well, 22 days later and I don't exactly remember why I added this. I think I saw some issue on Jenkins, let me remove it and see. Works fine locally.